src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -271,6 +274,36 @@ def joanie_product_widget_props(context):
     return json.dumps({"productId": product_id, "courseCode": course_code})
 
 
+@register.simple_tag()
+def get_categories_pages_have_faqs(course: Course) -> list[Page]:
+    """
+    Return a list with all pages have faq
```

Comment:
Replace comment with:
Return a categories pages that have FAQs and also have a page id.